luaa.c

```diff
@@ -327,14 +327,12 @@ luaA_pixbuf_to_surface(lua_State *L)
  * @treturn gears.surface A cairo surface as light user datum.
  * @treturn nil|string The error message, if any.
  * @staticfct load_image
+ * @deprecated awesome.pixbuf_to_surface
  */
 static int
 luaA_load_image(lua_State *L)
 {
-    /* TODO: Deprecate this function, Lua can use GdkPixbuf directly plus
-     * awesome.pixbuf_to_surface
-     */
-
+    luaA_deprecate(L, "awesome.pixbuf_to_surface");
```

Comment:
The suggestion for `awesome.pixbuf_to_surface` in that comment has since become outdated as well. I think `gears.surface` would be a better suggestion.

Reply:
`gears.surface` [uses](https://github.com/awesomeWM/awesome/blob/0f950cbb625175134b45ea65acdf29b2cbe8c456/lib/gears/surface.lua#L58) `awesome.pixbuf_to_surface` internally, and I would argue that it's better to put a function as close to the replaced one in functionality in `@deprecate`